rs/ethereum/types/src/address/mod.rs

```diff
@@ -31,6 +31,10 @@ impl Address {
     pub const fn new(bytes: [u8; 20]) -> Self {
         Self(bytes)
     }
+
+    pub const fn into_bytes(self) -> [u8; 20] {
```

Comment:
@ninegua @THLO This will be used in the EVM-RPC canister when converting to a Candid type